README.md

```diff
@@ -295,7 +295,7 @@ python examples/readme_example.py
 
 ## Learn more about workflows 
 
-* [Serverless Workflows Documentation](https://docs.tensorlake.ai/workflows/quickstart)
-* [Key programming concepts in Tensorlake Workflows](https://docs.tensorlake.ai/workflows/compute)
-* [Dependencies and container images in Tensorlake Workflows](https://docs.tensorlake.ai/workflows/images)
+* [Serverless Workflows Documentation](https://docs.tensorlake.ai/faqs/workflows-faq)
```

Comment:
Thank you. This needs to be https://docs.tensorlake.ai/applications/quickstart.
We changed the product name from workflows to applications.